core/metadata_vec.h

```diff
@@ -244,6 +237,16 @@ class MetaDataVec: public MetaData {
      */
     bool containsLabel(const MDLabel label) const override;
 
+    size_t firstRowId() const override;
+    size_t firstObject(const MDQuery&) const override;
```

Comment:
this method is removed, no?

Reply:
Nope, method with `MDQuery` as parameter is interntionally kept. Method withnout parameter was removed in other pull request.